### DISCUSSION TITLE
Is it possible to use ACF Blocks in this starter?

### DISCUSSION THREAD

**Discussion by W1KTOR3435** [colbyfayock/next-wordpress-starter#497] [Q&A]
Hi, currently I am looking for solution that will allow me to build page out of ACF blocks connected to Next.js. Is it possible to do it using your starter and WPGraphQL for Advanced Custom Fields plugin, because I couldn't find any example similar to what I want to achieve?

**Comment by KeriGerheart** [comment]
Hi, @W1KTOR3435 I'm not the author of the theme, but this [youtube video](https://www.youtube.com/watch?v=mMaw1LMClwQ&t=752s) from @colbyfayock might help you if you are looking to add ACF content to your posts. 

I am also using ACF fields on stand alone pages (as in, I have actual page files for them, they are not generated by a [slug].js file, since they don't use the WP content and have different layouts), I can give you some insight to the approach I took there if that would be helpful!
